style.css

```diff
@@ -121,4 +121,125 @@ h1
   {
     margin-right: 10px;
   }
+
+.top-hero-container
```

Comment:
Can you make sure you are using flex-box properly?

Reply:
i changed to auto